src/quizzes/chapter-2/2.2/basic-workflow-quiz.js

```diff
@@ -0,0 +1,35 @@
+const rawQuizdown = `
+---
+shuffleQuestions: true
+shuffleAnswers: true
+---
+
+# When printing a GitHub secret from a step in a workflow, what does it display as?
+
+1. [x] ***
+	> Good. This shows that a secret was being used without showing its actual sensitive contents. Note that the actual value of the secret *is* used, so in cases like password authentication, it will still work but just won't be displayed in the workflow log
+1. [ ] Nothing displays
+	> Not quite. This is usually indicative of the secret never being populated in the first place, meaning there is nothing to print at all
+1. [ ] Its actual value displays
+	> Thankfully this isn't the case. Imagine if your secret was holding authentication information for an organization, and it was publically visible in the workflow log!
+1. [ ] A random hex address relating to the secret, but not its actual value
+	> This wouldn't be a bad way to go about it, but isn't what actually happens
```

Comment:
Are you saying this wouldn't be a bad idea to have a hex number that actually _is_ the secret? Or just a hex number that is a _label_ for a secret but not some obfuscated value. If you mean the latter I that would be find if you mean the former I disagree

Reply:
I meant the latter 👍